frontend/src/components/features-style.css

```diff
@@ -0,0 +1,28 @@
+.card{
```

Comment:
I would rename the file to features.css and the .css implies its a style sheet. Also move the location of this file to a new folder under src named styles